server/applicationPattern/applicationPattern/onfModel/services/LogicalTerminationPointWithMappingServices.js

```diff
@@ -148,6 +149,33 @@ exports.findAndUpdateApplicationInformationAsync = function (logicalTerminationP
     });
 }
 
+/**
+ * @description This function finds an application by name and updates the http,
+ * operation and tcp client if require.
+ * @param {object} logicalTerminationPointConfigurationInput : is an instance of
+ * logicalTerminationPoint/ConfigurationInput class
+ * @return {Promise} object {LogicalTerminationPointConfigurationStatus} or null if the application is not found
+ **/
+exports.findAndUpdateApplicationInformationExcludingOldReleaseAndNewReleaseAsync = function (logicalTerminationPointConfigurationInput) {
+    return new Promise(async function (resolve, reject) {
+        let logicalTerminationPointConfigurationStatus = null;
+        try {
+            let applicationName = logicalTerminationPointConfigurationInput.applicationName;
+            let isApplicationExists = await httpClientInterface.isApplicationExistsExcludingOldReleaseAndNewRelease(
+                applicationName
+            );
+            if (isApplicationExists) {
+                logicalTerminationPointConfigurationStatus = await findAndUpdateLogicalTerminationPointInstanceGroupExcludingOldReleaseAndNewReleaseAsync(
+                    logicalTerminationPointConfigurationInput
+                );
+            }
```

Comment:
This function is going to be used just once, so I would suggest copying lines 164-171 directly to BaseServices.